### DISCUSSION TITLE
Explorer Restarts Randomly?

### DISCUSSION THREAD

**Discussion by ITDev93** [valinet/ExplorerPatcher#2384] [Q&A]
I have observed this, and I am not sure of the cause. 

Unfortunately, I cannot capture this as I do not know the trigger, but Explorer randomly restarts. I do not install preview updates or Insider; I only install vanilla updates...

Here is a console log:
![image](https://github.com/valinet/ExplorerPatcher/assets/22062468/3ebb113f-9987-4f36-bc32-269d6c627e77)

Am I the only one having this issue?

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application.

Please keep that EP console opened so you can see what happens when explorer crashes.

if you enabled the option register as file extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as file extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension
